CHANGELOG_PENDING.md

```diff
@@ -20,6 +20,7 @@ IMPROVEMENTS:
     - only process one block at a time to avoid starving
 - [crypto] Switch hkdfchachapoly1305 to xchachapoly1305
 - [common] bit array functions which take in another parameter are now thread safe
+- [p2p] \#2093 begin connecting to peers as soon a seed node provides them to you
```

Comment:
Let's create a markdown link to the issue and have it at the end.

Reply:
Not sure what you mean by "have it at the end". Do you mean make this a bug fix?

Reply:
Sorry I was ambiguous. Make the issue reference an actual link and have it at the end of the line, like this:

``` md
[p2p] begin connecting to peers as soon a seed node provides them to you ([#2093](https://github.com/tendermint/tendermint/issues/2093))
```

Reply:
:o thanks!